src/pages/RepoSearchPage/ReposSearchPage.tsx

```diff
@@ -49,28 +53,38 @@ const ReposSearchPage: React.FC = () => {
   }, []);
 
   return (
-    <div className="repo">
-      <div className="repo__search">
-        <Input
-          value={searchInput}
-          onChange={handleInput}
-          placeholder={"Введите название организации"}
-        />
-        <Button
-          isDisabled={isLoading}
-          onClick={getRepos}
-          inputValue={searchInput}
-        >
-          <SearchIcon />
-        </Button>
+    <context.Provider
+      value={{
+        list: reposArr,
+        isLoading: isLoading,
+        load: getRepos,
+      }}
+    >
+      <div className="repo">
+        <div className="repo__search">
+          <Input
+            value={searchInput}
+            onChange={handleInput}
+            placeholder={"Введите название организации"}
+          />
+          <Button
+            isDisabled={isLoading}
+            onClick={getRepos}
+            inputValue={searchInput}
+          >
+            <SearchIcon />
+          </Button>
+        </div>
+        <ul className="repo__list">
+          {!isLoading &&
+            reposArr.map((item: RepoItem) => (
+              <Link to={REPOS_ROUTE + `/:${item.name}`}>
+                <RepoTile key={item.id} item={item} onClick={() => {}} />
```

Comment:
`onClick` не нужен, вы и сами в него передаете пустой колбэк

```suggestion
                <RepoTile key={item.id} item={item} />
```